packages/tools/src/tools/annotation/PlanarFreehandROITool.ts

```diff
@@ -503,10 +499,12 @@ class PlanarFreehandROITool extends AnnotationTool {
         camera,
         spacingInNormalDirection
       );
-    } else {
+    } else if (viewport instanceof StackViewport) {
```

Comment:
I think this was fine, no need to change it, since we are using it on the VideoViewport, why do you need to add instanceof StackViewport